neqo-crypto/src/aead.rs

```diff
@@ -53,6 +53,11 @@ pub struct Aead {
     ctx: AeadContext,
 }
 
+// This allows fn expansion() to take &self.
+// The directive is placed here because
+// we cannot stack directives.
+// Placing it on top of #[must_use] doesn't work.
+#[allow(clippy::unused_self)]
```

Comment:
It seems like this lint did not exist for the toolchain our CI uses.
Not really sure if/when we want to bump versions, so I'll remove this directive for now.
Let's keep in mind we will have to add it again once we update our clippy/rustc versions in the CI